source/blog/2016-07-08-release-notes-6-36.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: Finagle 6.36 Release Notes
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Releases, Finagle, Util, Ostrich, Scrooge, TwitterServer
+---
+
+This is Finagle's first release only for Java 8 and Scala 2.11. Since previous release, the team
+has been focusing on Netty 4 migration as well as eliminating some technical debt.
+
+### Towards Netty 4
+
+The team is firing on all cylinders towards Netty 4 support in Finagle. This milestone brings
+*experimental* (not yet tested in production) Netty 4 support to Mux/ThriftMux, HTTP 1.1, and
```

Comment:
what do you think of calling it beta instead? we have prod usage for http/1.1 and mux.